build/rollup.config.js

```diff
@@ -3,13 +3,14 @@
 const path = require('path')
 const { babel } = require('@rollup/plugin-babel')
 const resolve = require('@rollup/plugin-node-resolve')
+const replace = require('@rollup/plugin-replace')
```

Comment:
Does popper have the string `NODE_ENV`? If not and if it's just for debugging we can set the env without any plugin. But maybe I'm missing something, so please shed some more light :)